docs/ruleset_functions/is_baseline_system_5.md

```diff
@@ -0,0 +1,50 @@
+# is_baseline_system_5  
+
+**Description:** Get either Sys-5, Sys-5b, or Not_Sys_5 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 5 (Package VAV with Reheat) or system 5b (system 5 with purchased heating).  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as either Sys-5, Sys-5b, or Not_Sys_5 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+- **terminal_unit_id_list**: list of terminal unit IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+- **zone_id_list**: list of zone IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+
+**Returns:**  
+- **is_baseline_system_5**: The function returns either Sys-5, Sys-5b, or Not_Sys_5 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 5 (Package VAV with Reheat) or system 5b (system 5 with purchased heating).   
+ 
+**Function Call:** 
+1. does_each_zone_have_only_one_terminal()    
+2. is_hvac_system_multizone()  
+3. is_hvac_sys_cooling_type_DX()
+4. is_hvac_sys_fan_sys_VSD()  
+5. is_hvac_sys_preheating_type_fluid_loop()
+6. is_hvac_sys_preheat_fluid_loop_attached_to_boiler()
+7. is_hvac_sys_preheat_fluid_loop_purchased_heating()  
+8. are_all_terminal_heat_sources_hot_water()  
+9. are_all_terminal_heating_loops_attached_to_boiler()  
+10. are_all_terminal_heating_loops_purchased_heating()  
+11. are_all_terminal_cool_sources_none_or_null() 
+12. are_all_terminal_fans_null()  
+13. are_all_terminal_types_VAV()
+14. is_there_only_one_hvac_sys_cooling_system ()  
+
+ 
+## Logic:    
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Set is_baseline_system_5 = Not_Sys_5: `is_baseline_system_5 = "Not_Sys_5"`    
+- Check that there is no heating system, if there is none then carry on: `if len(hvac_b.heating_system) == Null or hvac_b.heating_system[0].heating_system_type == "NONE":`    
+    - Check that there is one preheat system per G3.1.3.19, if there is then carry on: `if Len(hvac_b.preheat_system) == 1:`   
```

Comment:
@claperle I believe I might have asked this before as well. For system type 5 (or 6-8) do we assume that central  heating = NULL? I always thought that central heating is also the same as the reheat source.

Reply:
@supriyagoel  My understanding is it is because they have a preheat coil they would not have heating coil. They would not have a heating coil since they have the terminal reheat. Note that per G3.1.3.19 systems 5 through 8 have preheat coils.

Reply:
Comment closed, discussed as group on call. There should be preheat, reheat and that's it for heating.